### DISCUSSION TITLE
Is there a way for me to adjust the contrast on ignored + non-project files

### DISCUSSION THREAD

**Discussion by adampatterson** [one-dark/jetbrains-one-dark-theme#298] [General]
In my old age I am finding these files hard to read. Is there any way I can adjust the background or ignored file names?

![image](https://user-images.githubusercontent.com/31537/222261293-3f7d409a-d0c1-426c-aa03-616142339193.png)

**Comment by Unthrottled** [comment]
Yes!

You are looking for the ['ignored' color file status color](https://www.jetbrains.com/help/idea/file-status-highlights.html). tldr: Settings -> Version Control -> File Status Colors -> Ignored.

<img width="1094" alt="Screenshot 2023-03-02 at 6 47 44 PM" src="https://user-images.githubusercontent.com/15972415/222603841-8a70cdf9-0814-4b2a-81c4-de8c258636f1.png">

**Reply by adampatterson** [reply]
Thanks so much! 

Would you happen to know where I can adjust the background colour?

**Reply by Unthrottled** [reply]
File status colors is what you are looking for
https://www.jetbrains.com/help/idea/configuring-scopes-and-file-colors.html#new-color

**Comment by adampatterson** [comment]
Thanks @Unthrottled !
